.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Use indentation of two(2) spaces for this file and any other file.

Reply:
All done @MuhanguziDavid